tests/functional/aws-node-sdk/test/multipleBackend/objectCopy/objectCopy.js

```diff
@@ -13,6 +13,7 @@ const { describeSkipIfNotMultiple, awsS3, memLocation, awsLocation,
     require('../utils');
 const bucket = 'buckettestmultiplebackendobjectcopy';
 const bucketAws = 'bucketawstestmultiplebackendobjectcopy';
+const bucketEncrypted = 'bucketenryptedtestmultiplebackendobjectcopy';
```

Comment:
can we call this awsserversideencryptionbucket?  It's confusing to call it bucketEncrypted since that implies our bucket encryption.